necroHeal.lic

```diff
@@ -0,0 +1,43 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placement comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status

```

Comment:
The one goal of `safe-room` is to heal you up. This sounds like an extension of that.

There's even a `!DRStats.necromancer?` check to avoid taking you to Dokt - I imagine we could just run `hunting-buddy(heal)` in that case.

`Safe-room` already runs as part of `training-manager`, so you wouldn't need it as an 'after' script either.